Config.xml

```diff
@@ -0,0 +1,10 @@
+<xml/>
+<InstrumentationEngineConfiguration>
```

Comment:
>InstrumentationEngineConfiguration [](http://example.com/codeflow?start=1&length=34)

Can this be moved to the "test" folder or rename the folder to something like "TestConfig"?

Reply:
This was accidentally added from the output of a test that I ran. I will remove it.